include/deal.II/distributed/tria.h

```diff
@@ -959,16 +984,6 @@ namespace parallel
       std::vector<types::global_dof_index> coarse_cell_to_p4est_tree_permutation;
       std::vector<types::global_dof_index> p4est_tree_to_coarse_cell_permutation;
 
-      /**
-       * dummy settings
-       */
-      enum Settings
-      {
-        default_setting = 0x0,
-        mesh_reconstruction_after_repartitioning = 0x1,
-        construct_multigrid_hierarchy = 0x2
-      };
-
```

Comment:
Any reason for moving this?

Reply:
yes, it should be defined before constructor, right?

Reply:
Yes, you're right.